BuildScript/BuildTargets/Testing.targets

```diff
@@ -115,9 +116,23 @@
     <!-- Run tests -->
 
     <PropertyGroup>
-      <_testName>$(_testAssemblyFileName).$(ConfigurationID).$(_platform).$(_executionRuntime).$(_targetRuntime)</_testName>
+      <_commaSeparatedTestCategoriesToExclude>$(TestCategoriesToExclude.Replace(';', ','))</_commaSeparatedTestCategoriesToExclude>
+      <_mergedTestCategoriesToExclude>$(_commaSeparatedTestCategoriesToExclude),$(_excludeCategoriesInTestConfiguration)</_mergedTestCategoriesToExclude>
```

Comment:
am wondering if it would be easier to read/maintain if we made sure the property is empty if there's no categories instead of containing a single comma.